glusterd2/commands/volumes/volume-edit.go

```diff
@@ -63,6 +68,12 @@ func volumeEditHandler(w http.ResponseWriter, r *http.Request) {
 			volinfo.Metadata[key] = value
 		}
 	}
+
+	metadataSize := volinfo.MetadataSize()
```

Comment:
Use the function call directly in if condition, there's no need for extra variable.